Sync/SyncStateMachine.swift

```diff
@@ -25,14 +25,18 @@ private let LocalEngines: [String] = TogglableEngines + ["clients"]
 // Names of collections which will appear in a default meta/global produced locally.
 // Map collection name to engine version.  See http://docs.services.mozilla.com/sync/objectformats.html.
 private let DefaultEngines: [String: Int] = [
+    "bookmarks": 2,
     "clients": ClientsStorageVersion,
     "history": HistoryStorageVersion,
     "tabs": TabsStorageVersion,
     // We opt-in to syncing collections we don't know about, since no client offers to sync non-enabled,
     // non-declined engines yet.  See Bug 969669.
```

Comment:
@justindarc If you think it'll be helpful, would you mind cribbing [this comment](https://github.com/mozilla-mobile/firefox-ios/pull/4549/files#diff-996382204f85c037854c773b09b618d6R33), too?